pages/users/[id]/trades/purchased.tsx

```diff
@@ -239,9 +240,10 @@ const TradePurchasedPage: NextPage = () => {
                           aria-label="external link"
                           as={Link}
                           href={
-                            blockExplorer(item.asset?.chainId).transaction(
-                              item.transactionHash,
-                            ) || '#'
+                            blockExplorer(
+                              CHAINS,
+                              item.asset?.chainId,
+                            ).transaction(item.transactionHash) || '#'
```

Comment:
Do we still need passing `CHAINS` if we revert this function like the way I've mentioned? Maybe I'm missing something. But revert if possible.
```suggestion
                            blockExplorer(item.asset?.chainId).transaction(
                              item.transactionHash,
                            ) || '#'
```

Reply:
As I described in the other comment, just `blockExplorer` won't be able to use other hooks it it's not a hook itself